docker/1.5.1/py2/dockerd-entrypoint.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
Where is this script used?

Reply:
Seems like a legacy file from 1.4.1 and not in use anymore. Will remove this

Reply:
Seems this file is used by Dockerfile.gpu for 1.4.1 py2. Other dockerfiles are using `dockerd-entrypoint.py`. Will remove this